js_modules/dagster-ui/packages/ui-components/src/components/Intent.tsx

```diff
@@ -0,0 +1 @@
+export type Intent = 'none' | 'primary' | 'success' | 'warning' | 'danger';
```

Comment:
Big fan of making these string unions, I feel like proper TS enums are overrated.

Reply:
Definitely.